src/components/Selections.vue

```diff
@@ -12,6 +13,58 @@ import { store } from '../store';
 
 export default {
 	name: 'Selections',
+
+	data() {
+		return {
+			// stores success text to be displayed on selection send
+			successText: ''
+		}
+	},
+	methods: {
+		back() {
+			//go back to previous page
+			this.$router.push('pie');
+		},
+		saveSelections(){
+
+			var personName = this.$store.getters.selectedPerson.name;
+			var pieName = this.$store.getters.selectedPie.name;
+
+			// format the request data into an acceptable format
+			var tempSelection = {
+				name: personName,
+				pie: pieName
+			};
+
+			// prepare and send the write request
+			var xhr = new XMLHttpRequest();
+
+			// The server URL can be changed in the store
+			xhr.open('POST', this.$store.getters.serverURL+'/writeSelections.php', true);
```

Comment:
I had to add /backend to the /writeSelections URL, as I was getting '[404]: /writeSelections.php - No such file or directory' error

Reply:
ah, I see what's happened. If you're running server.sh from the root directory, then all the backend stuff will be in localhost:80/backend. I had always been running it from inside the backend/ directory so all the backend stuff was in the root of localhost:80. I'll change the serverURL variable to be localhost:80/backend permanently, because I think in the readme I wrote that the server should be started from the root directory :)

Thanks for reviewing this!